src/utils/DataProcessingUtils.js

```diff
@@ -0,0 +1,479 @@
+
```

Comment:
good ol' data processing utils back again 😄 this file looks just like the `DataProcessingUtils` from Milestone. was there anything you changed? doesn't seem like it to me.

Reply:
You can track changes to this file here #13. The short of it is that it's going to also accept plain js instead of strictly immutable objects. The motivation for this change is that the lib should be flexible to handle both situations. In addition, since the formData should typically not need to be handled externally, its totes fine to skip the whole immutable bit esp since RJSF does not accept immutable collections anyway. There's just a bunch of to/fromJS() being called and it tends to slow things down.

Reply:
oh and it needs to change a little so that it's not so reliant on the shape of the edm in the consuming app's redux store.